enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -0,0 +1,157 @@
+package podman
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/commandutil"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/container"
+	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
+	"github.com/buildbuddy-io/buildbuddy/server/util/background"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+	"github.com/buildbuddy-io/buildbuddy/server/util/random"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+)
+
+var (
+	containerFinalizationTimeout = 10 * time.Second
+)
+
+// podmanCommandContainer containerizes a command's execution using a Podman container.
```

Comment:
love the go style comments :P If you added them to all public methods in this piece, I'd be so happy!

Reply:
The interface commanderContainer has all the comments for the public methods.